celements-user-manager/web-module/src/main/webapp/resources/celRes/userAdmin/table.css

```diff
@@ -0,0 +1,145 @@
+    padding: 5px 0px;
+    margin: 0px 0px;
+    font-weight: bold;
+}
+
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table ul.struct_table_data {
+    display: flex;
+    flex-flow: column;
+    padding: 0px 0px;
+    margin: 0px 0px;
+}
+
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table ul.struct_table_data .struct_table_row:nth-child(odd){
+    background-color: whitesmoke;
+}
+
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table ul li.struct_table_row {
+    display: flex;
+    flex-flow: row wrap;
+    align-items: flex-start;
+    padding: 2px 0px;
+    margin: 1px 0px;
+}
+
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table ul.struct_table_data li.struct_table_row:hover,
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table ul.struct_table_data li.struct_table_row.open {
+    background-color: lightgrey;
+}
+
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table div.column_arrow {
+    flex: 0 0 20px;
+    text-align: center;
+}
+
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table li.struct_table_row div.column_arrow {
+    -webkit-transition: transform 0.5s linear;
+    -moz-transition: transform 0.5s linear;
+    -o-transition: transform 0.5s linear;
+    transition: transform 0.5s linear;
+    transform: rotate(0deg);
+    content: "\f0da";
+  }
+
+  div.userAdmin div.searchUser #searchoutput cel-table.struct_table li.struct_table_row.open div.column_arrow {
+    -webkit-transition: transform 0.5s linear;
+    -moz-transition: transform 0.5s linear;
+    -o-transition: transform 0.5s linear;
+    transition: transform 0.5s linear;
+    transform: rotate(90deg);
+}
+
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table div.column_active {
+    flex: 0 0 22px;
+    text-align: center;
+}
+
+.fa-user-slash {
+    color: red;
+}
+
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table .column_name {
+    flex: 1 0 150px;
+}
+
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table .column_firstname{
+    flex: 1 0 100px;
+}
+
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table .column_email{
+    flex: 1 0 350px;
+}
+
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table .column_roles {
+    flex: 2 0 120px;
+}
+
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table .column_edit, 
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table .column_delete {
+    flex: 0 0 20px;
+}
+
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table div.column_edit a.icon-link, 
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table div.column_delete a.icon-link {
+    text-decoration: none;
+    color: black;
+}
+
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table div.column_edit a.icon-link:hover, 
+div.userAdmin div.searchUser #searchoutput cel-table.struct_table div.column_delete a.icon-link:hover {
+    color: #156eb9;
```

Comment:
I see this color repeated many times across multiple CSS. better define a variable.
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties